Formula/tezos-accuser-011-PtHangz2.rb

```diff
@@ -27,15 +27,14 @@ class << self
 
   bottle do
     root_url "https://github.com/serokell/tezos-packaging/releases/download/#{TezosAccuser011Pthangz2.version}/"
```

Comment:
Are bottles independent from architectures?

I see that currently released bottles include in their name the package version and the macOS version used, but no architecture.

Reply:
Seems like brew is able to handle this, there are `big_sur` and `arm64_big_sur`, see [this formula](https://github.com/Homebrew/homebrew-core/blob/master/Formula/acmetool.rb#L12) as an example

Reply:
> Are bottles independent from architectures?

So, the answer is no

Reply:
Never mind, I see from [the documentation](https://docs.brew.sh/Bottles) that architecture and OS version are handled together, so this `root_url` still works.